conf/app.routes

```diff
@@ -62,6 +62,12 @@ POST       /when-did-health-issue-happen                   controllers.HealthRea
 POST       /when-did-health-issue-happen/change            controllers.HealthReasonController.onSubmitForWhenHealthReasonHappened(mode: Mode = CheckMode)
 
 
+# Oth Reasonable Excuse Journey
+GET        /when-did-you-become-unable              controllers.OtherReasonController.onPageLoadForWhenDidBecomeUnable(mode: Mode = NormalMode)
```

Comment:
Not sure on the naming of the URL - this would perhaps be problematic for an agent view as well. Perhaps `/when-inability-to-manage-account-happened`